kubectl-rook-ceph.sh

```diff
@@ -245,6 +249,21 @@ function run_rook_cr_status() {
   fi
 }
 
+function run_purge_osd() {
+  if [ "$#" -eq 1 ]; then
+    mon_endpoints=$($TOP_LEVEL_COMMAND --namespace "$ROOK_CLUSTER_NAMESPACE" get cm rook-ceph-mon-endpoints -o jsonpath='{.data.data}' | cut -d "," -f1)
+    ceph_secret=$($TOP_LEVEL_COMMAND --namespace "$ROOK_OPERATOR_NAMESPACE" exec deploy/rook-ceph-operator -- cat /var/lib/rook/"$ROOK_CLUSTER_NAMESPACE"/client.admin.keyring | grep "key" | awk '{print $3}')
+    $TOP_LEVEL_COMMAND --namespace "$ROOK_OPERATOR_NAMESPACE" exec deploy/rook-ceph-operator -- sh -c "export ROOK_MON_ENDPOINTS=$mon_endpoints \
+      ROOK_CEPH_USERNAME=client.admin \
```

Comment:
Will this be affected by our current desires to change the client? Should we change something here for forwards compatibility?

Reply:
I believe not, in the future we change the client we'll just have to change the client name here.

Reply:
For backward compatibility, we will need to keep the client.admin, so I don't see a problem if we keep this for now. Ideally, perhaps we even create a `client.krewplugin` user in the future after that PR, but we can decide later if it's worth it.